tests/virt_autotest/install_package.pm

```diff
@@ -68,10 +69,10 @@ sub install_package {
         }
     }
 
-    ###SLE-12-SP4 arm64 installation has no KVM role selection
-    if (($repo_0_to_install =~ /SLE-12-SP4/m) && check_var('ARCH', 'aarch64')) {
-        zypper_call("--gpg-auto-import-keys ref",         180);
-        zypper_call("in -t pattern kvm_server kvm_tools", 300);
+    ###Install KVM role patterns form aarch64
```

Comment:
do you mean *from* or *for*? The former comment described the reason better. Now it's stating the obvious. Why is this only relevant for the remote backend now?

Reply:
Because we only use remote backend in terms of this test module.

Reply:
The comment unfortunately is still "stating the obvious" and not the reason as was done in before. Please reconsider the comment or leave out that change of the PR for now.